sdk/helper/ldaputil/ldap.go

```diff
@@ -16,16 +14,11 @@ func NewLDAP() LDAP {
 // LDAP provides ldap functionality, but through an interface
 // rather than statically. This allows faking it for tests.
 type LDAP interface {
-	Dial(network, addr string) (Connection, error)
-	DialTLS(network, addr string, config *tls.Config) (Connection, error)
+	Dial(addr string, opts ...ldap.DialOpt) (Connection, error)
```

Comment:
Since this is no longer backwards compatible with the previous interface, we should consider renaming `Dial` to `DialURL` to match the library method.